src/main/java/site/sonisori/sonisori/auth/jwt/JwtUtil.java

```diff
@@ -103,4 +111,15 @@ public boolean validateAccessToken(String accessToken) {
 			throw new JwtException(e.getMessage());
 		}
 	}
+
+	public Long getUserId(String token) {
+		Claims claims = extractClaims(token);
+		return Long.parseLong(claims.getSubject());
+	}
+
```

Comment:
이 부분을 getUsername함수로 바꾸면 될 것 같네요!
```suggestion
	public Long getUsername(String token) {
		Claims claims = extractClaims(token);
		return claims.get("username", String.class);
	}

```